feature/session/src/main/java/io/github/droidkaigi/confsched2019/session/ui/widget/TimetableCurrentTimeLabelDecoration.kt

```diff
@@ -0,0 +1,110 @@
+package io.github.droidkaigi.confsched2019.session.ui.widget
+
+import android.content.Context
+import android.graphics.Canvas
+import android.graphics.Color
+import android.graphics.CornerPathEffect
+import android.graphics.Paint
+import android.graphics.Path
+import android.graphics.Rect
+import androidx.core.content.ContextCompat
+import androidx.recyclerview.widget.RecyclerView
+import com.soywiz.klock.DateFormat
+import com.soywiz.klock.DateTimeTz
+import com.soywiz.klock.hours
+import com.xwray.groupie.GroupAdapter
+import io.github.droidkaigi.confsched2019.session.R
+
+class TimetableCurrentTimeLabelDecoration(
+    private val labelWidth: Float,
```

Comment:
Why do you use parameter for it? ⛱ 
I think you can just use a field like this.

```
class TimetableCurrentTimeLabelDecoration(context: Context, groupAdapter: GroupAdapter<*>) {
    private val labelWidth: Float = context.resources.getDimension(R.dimen.tabular_form_time_label_width)
```